package.json

```diff
@@ -60,8 +60,8 @@
   "devDependencies": {
```

Comment:
Can you add `{ "engines" : { "node" : ">=6.13", "npm": ">= 5.7.1" } }` to document the recommended node and npm versions for docs?